tasks/main.yml

```diff
@@ -83,6 +83,7 @@
     from_port: "{{ item.from_port | default('') }}"
     to_port: "{{ item.to_port | default('') }}"
     protocol: "{{ item.protocol | default('any') }}"
+    route: "{{ item.route | default('no') }}"
```

Comment:
Are you sure that `no` is the default. Or should it be `omit` or even `''`? For me it would be easier if I have documentation or code to verify that this does not break anything.

Reply:
I think you are right to think de default is `omit`. I think I misread the documentation last time and mistakenly thought the parameter defaulted to `'no'`. I know for sure that `''` does not work as it failed some builds on travis.

I will investigate and update my pull request.